go/ct/rlz/domains.go

```diff
@@ -422,3 +422,69 @@ func removeDuplicatesGeneric[T comparable](slice []T) []T {
 
 	return result
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// Delegation designator domain
+
+// DelegationDesignatorState represents the state of a delegation designator in
+// the code of an account.
+// This type works together with ConstraintDelegationDesignator condition to model
+// EIP-7702 behavior.  see https://eips.ethereum.org/EIPS/eip-7702
+type DelegationDesignatorState = uint64
+
+const (
+	NoDelegationDesignation DelegationDesignatorState = iota
+	WarnDelegationDesignation
+	ColdDelegationDesignation
+)
+
+func DelegationDesignatorName(dd DelegationDesignatorState) string {
```

Comment:
This should probably be implement as
```suggestion
func (dd DelegationDesignatorState) String() string {
```

Reply:
https://github.com/0xsoniclabs/tosca/pull/37/commits/90ca95831b05441005e8f0d5c1168f19f48b3b34